server/src/pages/homepage/view.js

```diff
@@ -79,6 +79,8 @@ class Body extends React.Component {
       <reactruntime.BodyTemplate {...this.props}>
         <HomePageHeader isOwner={this.props.showMyShots} isFirefox={this.props.isFirefox}
           hasFxa={this.props.hasFxa} />
+        {this.props.showMyShots ? null : <a href="/api/fxa-oauth/login/">Login</a>}
```

Comment:
Obviously this is the hackiest of all links, and probably the logic (`this.props.showMyShots`) is wrong too, and is *definitely* named incorrectly.